index.html

```diff
@@ -3,184 +3,196 @@
 active: home
 ---
 
-<div style="top:0;position:absolute;display: inline;">
-<!-- Floating scroll-down button -->
-    <button id="carousel-scroll-down-button"  aria-label="Scroll to the end/bottom of the image carousel">
-         <svg xmlns="http://www.w3.org/2000/svg"
-              viewBox="-20 -20 355 155">
-            <path id="carousel-scroll-down-button-path"
-                    fill="none" stroke="white" stroke-width="25" stroke-linecap="round" opacity=".9"
-                    d="M 0 0 C 0 0 141.4 86.1 141.4 86.1 C 141.4 86.1 282.8 0 282.8 0" />
-         </svg>
-    </button>
-</div>
-
-<div id="slider">
-    <div>
-        <picture>
-            <source srcset="https://assets.csh.rit.edu/pubsite/welcome_back_large.webp" type="image/webp" />
-            <source srcset="https://assets.csh.rit.edu/pubsite/welcome_back_large.jpeg" type="image/jpeg" />
-            <img src="https://assets.csh.rit.edu/pubsite/welcome_back_large.jpeg" alt="A group of CSHers standing in front of a lake" />
-        </picture>
-    </div>
-    <div>
-        <picture>
-            <source srcset="https://assets.csh.rit.edu/pubsite/server_room.webp" type="image/webp" />
-            <source srcset="https://assets.csh.rit.edu/pubsite/server_room.jpeg" type="image/jpeg" />
-            <img src="https://assets.csh.rit.edu/pubsite/server_room.jpeg" alt="A CSH member stands at a terminal in the CSH server room" />
-        </picture>
-    </div>
-    <div>
-        <picture>
-            <source srcset="https://assets.csh.rit.edu/pubsite/rock.webp" type="image/webp" />
-            <source srcset="https://assets.csh.rit.edu/pubsite/rock.jpeg" type="image/jpeg" />
-            <img src="https://assets.csh.rit.edu/pubsite/rock.jpeg" alt="CSHers crowd around a rock painted red with a CSH logo on it in white" />
-        </picture>
+<section class="top-banner-section">
+    <div class="banner-image-div">
+    <img class="banner-image" src="assets/images/lounge.png" alt="Banner Image" />
```

Comment:
The alt should probably be a description of the image content, shouldn't it?

Reply:
Resolved